packages/core/src/__integrationtests__/Balance.spec.ts

```diff
@@ -159,14 +160,16 @@ describe('When there are haves and have-nots', () => {
   it('should be able to make multiple transactions at once', async () => {
     const listener = jest.fn()
     listenToBalanceChanges(faucet.address, listener)
-    await Promise.all([
-      getTransferTx(richieRich.address, EXISTENTIAL_DEPOSIT).then((tx) =>
-        submitExtrinsicWithResign(tx, faucet)
-      ),
-      getTransferTx(stormyD.address, EXISTENTIAL_DEPOSIT).then((tx) =>
-        submitExtrinsicWithResign(tx, faucet)
-      ),
-    ])
+
+    const api = await BlockchainApiConnection.getConnectionOrConnect()
+    const batch = api.tx.utility.batchAll(
+      await Promise.all([
+        getTransferTx(richieRich.address, EXISTENTIAL_DEPOSIT),
+        getTransferTx(stormyD.address, EXISTENTIAL_DEPOSIT),
+      ])
+    )
+    await submitExtrinsic(batch, faucet)
```

Comment:
HA, yeah good question. Since it is using `batch` now I guess it is still possible to make multiple transactions at once? Actually even better now. Just not from two devices, what we disregard now.